app/portainer/rbac/views/roles/roles.html

```diff
@@ -2,7 +2,7 @@
 
 <div class="row">
   <div class="col-sm-12">
-    <roles-datatable title-text="Roles" title-icon="fa-file-code" dataset="$ctrl.roles" table-key="roles"></roles-datatable>
+    <roles-datatable title-text="Roles" title-icon="file" dataset="$ctrl.roles" table-key="roles"></roles-datatable>
```

Comment:
Icon here is wrong, should use custom icon `file-code` according to our [icon guide](https://portainer-style-guide.webflow.io/)
![image](https://user-images.githubusercontent.com/12197793/182049010-080d9347-4c25-453a-ac11-06c7f3692cae.png)